_layouts/styleguide.html

```diff
@@ -9,8 +9,7 @@
   <body class="page-{{ page.title | slugify }}">
 
   	{% include navbar.html %}
-
-    <aside class="sidenav">
+    <aside class="sidenav sticky" style="top: 5rem; margin-bottom: 457px;">
```

Comment:
I based this on the [the example from `stickyfill`](https://github.com/wilddeer/stickyfill/blob/master/test/index.htm#L119). In this instance I think it makes more sense for them to be inline so that it is the ultimate in hierarchy for defining this interactivity. 💪